tailwind.config.ts

```diff
@@ -1,18 +1,33 @@
-import type { Config } from "tailwindcss";
+// import type { Config } from "tailwindcss";
 
-export default {
+// export default {
```

Comment:
Do we really need to change all default Tailwind config? Lets do that in separate tasks. A single PR should ideally bring in the minimal changes required to complete the task. Additional "good to have" changes should be done in separate PRs